Dockerfile

```
@@ -1,21 +1,20 @@
-FROM node:12-alpine AS build
+FROM node:lts-alpine AS build
```

```suggestion
FROM node:14-alpine AS build
```

Could we change these? Manually controlling the version should avoid us a bunch of breakage down the line, as explained.